### PR TITLE
Add Snowflake and Dremio blogs and link recent talks

### PR DESCRIPTION
**Pull request by danielcweeks**

### CODE REVIEW

**Review by samredai — APPROVED** [review]

File: docs/common/community/talks.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ Here is a list of talks and other videos related to Iceberg.
 ### [Expert Roundtable: The Future of Metadata After Hive Metastore](https://www.youtube.com/watch?v=7_Pt1g2x-XE)
 **Date**: November 15, 2021, **Authors**: Lior Ebel, Seshu Adunuthula, Ryan Blue & Oz Katz
 
+### [Presto and Apache Iceberg: Building out Modern Open Data Lakes](https://www.youtube.com/watch?v=OJQHVPChYHw)
+**Date**: November 10, 2021, **Authors**: Daniel Weeks, Chunxu Tang
```
Comment (samredai) [inline docs/common/community/talks.md:29]: I noticed the youtube video was posted on Dec 21st so is Nov 10th the date the actual talk took place? I think either one is fine but thought I would ask.

**Review by rdblue — APPROVED** [review]

**Comment by rdblue** [comment]
@samredai, do we sync this over to the iceberg-docs repo?

**Comment by samredai** [comment]
@rdblue Yes since we wouldn't want to wait until the next version release for this to get out, we should merge this commit into the main branch on the iceberg-docs repo where it will auto-deploy.

**Comment by rdblue** [comment]
Probably good to consider moving these PRs over to the docs repo then.

**Pull request merged**